agresso/handlers.py

```diff
@@ -0,0 +1,55 @@
+import json
+from datetime import datetime
+
+from aws_xray_sdk.core import patch_all, xray_recorder
+from okdata.aws.logging import logging_wrapper
+from okdata.sdk.data.dataset import Dataset
+from okdata.sdk.data.upload import Upload
+from requests.exceptions import HTTPError
+
+from agresso.service import (
+    get_budget,
+    get_budget_descriptions,
+    get_general_ledger,
+    get_user_log_contractor_invoices,
+    get_workflow_comment_parked,
+    get_workflow_contrator_invoices,
+)
+
+patch_all()
+
+
+def import_dataset(dataset_id, data):
+    print(f"Importing {len(data)} entries to dataset '{dataset_id}'...")
```

Comment:
Burde vi bruke `logging` fremfor `print`, eller har det ikke noe å si for logge-opplegget vårt (husker ikke helt jeg)?

Reply:
Det skal ikke ha noe å si, vanlig `print` kommer inn i loggene.